src/webportal/src/app/job-submission/components/tools/tensorboard.jsx

```diff
@@ -23,27 +21,47 @@ const style = {
   },
 };
 
+const generateDefaultTensorBoardExtras = port => {
+  const tensorBoardExtras = {
+    plugin: 'tensorboard',
+    parameters: {
+      port: port,
+      logdir: {
+        path: TENSORBOARD_LOG_PATH,
+      },
+    },
+  };
+  return tensorBoardExtras;
+};
+
 export const TensorBoard = props => {
-  const { jobData, taskRoles, extras, onChange } = props;
+  const { extras, onChange } = props;
+
+  // TensorBoard will use random port in [10000, 10100)
+  const tensorBoardPort = useMemo(
+    () => Math.floor(Math.random() * 100 + 10000),
```

Comment:
the range is small for pseudo random number

Reply:
Change to [10000, 15000). If this node running 32 tensorBoard, the possibility of ports conflict is about 0.1004